app/src/app/views/stix/collection/collection-import/collection-import-workflow/collection-import.component.ts

```diff
@@ -217,16 +223,18 @@ export class CollectionImportComponent implements OnInit {
                     this.loadingStep2 = true;
                     setTimeout(() => { //make sure the loading icon renders before the parsing/writing
                         let newBundle = JSON.parse(JSON.stringify(this.collectionBundle)); //deep copy
-                        let objects = []
-                        // filter objects to selected or unchanged
-                        for (let object of newBundle.objects) {
-                            if (this.unchanged_ids.includes(object.id) || this.select.selected.includes(object.id)) {
-                                // object is selected or unchanged
-                                objects.push(object);
+                        if (!force) {
+                            let objects = []
+                            // filter objects to selected or unchanged
+                            for (let object of newBundle.objects) {
+                                if (this.unchanged_ids.includes(object.id) || this.select.selected.includes(object.id)) {
+                                    // object is selected or unchanged
+                                    objects.push(object);
+                                }
                             }
+                            newBundle.objects = objects;
                         }
```

Comment:
Is there a reason that filtering of collection contents is not taken into account if they are forcing the import? The user's selection of which objects they want to import is still relevant.